modules/connecttest.lua

```diff
@@ -26,6 +26,13 @@ local SDL = require('SDL')
 local exit_codes = require('exit_codes')
 local load_schema = require('load_schema')
 
+local hmiAdapter
+if config.hmiAdapterType == "Remote" then
+  hmiAdapter = require('remote_hmi_adapter_connection')
+else if config.hmiAdapterType == "WebSocket" then
+  hmiAdapter = require('websocket_connection')
+end
```

Comment:
would be nice notify user in case no one variant match (typo or etc.)

Reply:
@KVGrygoriev Wrong HMI adapter handling was added (see f406fd26b2ba8306caa87ead9f83a9133efd7ca2)